domains/transfer/Transfer.API/API/Claiming/Startup.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using API.Claiming.Api.Repositories;
+using API.Claiming.Automation;
+using API.Claiming.Automation.Services;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace API.Claiming;
+
+public static class Startup
+{
+    public static void AddClaimServices(this IServiceCollection services)
+    {
+        services.AddScoped<IClaimAutomationRepository, ClaimAutomationRepository>();
+
+        services.AddScoped<IClaimService, ClaimService>();
+        services.AddScoped<IProjectOriginWalletService, ProjectOriginWalletService>();
+        services.AddScoped<Random>();
```

Comment:
I find it quite surprising to see the .NET class Random being registered in the DI. If you want to do it that way, I would prefer that you create a new class - let's call it "Shuffler" - that wraps an instance of Random. The "Shuffler" class is registered in DI, not Random.